docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -415,3 +421,31 @@ After logging out of `tsh`, you can log back in without specifying
 
 (!docs/pages/includes/sso/loginerrortroubleshooting.mdx!)
 
+### Teams Not Mapping to Roles
+
+If you are not seeing teams mapping to roles confirm you 
+are using the slug value of the organizations and teams in the connector.
+The GitHub Web application URIs and GitHub API can provide the slug values.
+
+Example with team name `My Team`:
+```yaml
+  teams_to_roles:
+    - organization: ORG-NAME
+      roles:
+        - access
+      team: My Team
+```
+
+Navigate to the team `My Team`. The url `https://github.com/orgs/ORG-NAME/teams/my-team`
```

Comment:
```suggestion
Navigate to the team `My Team`. The URL `https://github.com/orgs/ORG-NAME/teams/my-team`
```

URL is an acronym. Capitalize it.

Also, we should probably say `my-org` as well instead of `ORG-NAME`.